sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -482,6 +482,17 @@ object JdbcUtils extends Logging with SQLConfHelper {
           row.update(pos, null)
         }
 
+    case TimestampNTZType =>
+      (rs: ResultSet, row: InternalRow, pos: Int) =>
+        val t = rs.getTimestamp(pos + 1)
+        if (t != null) {
+          val micros = DateTimeUtils.millisToMicros(t.getTime) +
```

Comment:
to make it more readable, can we add `DateTimeUtils.fromJavaTimestampNoRebase`?

Reply:
OK